skglm/solvers/group_bcd_solver.py

```diff
@@ -131,9 +129,7 @@ def _bcd_epoch(X, y, w, Xw, datafit, penalty, ws):
         grad_g = datafit.gradient_g(X, y, w, Xw, g)
 
         w[grp_g_indices] = penalty.prox_1group(
-            old_w_g - grad_g / lipschitz_g,
```

Comment:
same here, can you revert them ? And find the root cause so that it does not happen again